src/rgw/rgw_log.cc

```diff
@@ -226,6 +225,15 @@ static void log_usage(struct req_state *s, const string& op_name)
   utime_t ts = ceph_clock_now();
 
   usage_logger->insert(ts, entry);
+
+  if (s->cct->_conf->rgw_enable_usage_log_at_subuser_level &&
+      ! s->auth.identity->get_subuser_name().empty()) {
+    string sname = u + ":" + s->auth.identity->get_subuser_name();
+    ldout(s->cct, 5) << "log subuser entry: " << sname << dendl;
+    rgw_usage_log_entry sentry(sname, p, bucket_name);
```

Comment:
Oops, possible conflict with multi-tenancy. See `rgw_usage_log_entry` and its constructors. We might need to extend it to carry subuser info as well.